.python-version

```diff
@@ -0,0 +1 @@
+3.8.12
```

Comment:
you should be able to use sentry's pypi -- we have wheels for confluent-kafka up to 3.11

`--index-url https://pypi.devinfra.sentry.io/simple`

Reply:
actually you already have that -- what error message are you hitting? https://github.com/getsentry/relay/blob/5e161d17f9b5af3468b1b044cf60a356cad63ab2/requirements-dev.txt#L1

Reply:
That's there already: https://github.com/getsentry/relay/blob/7bcda764a0c31b42d172cf8afa1a991cb3c69da0/requirements-dev.txt#L1

However, running with python 3.11.3, we receive this failure:

```
Successfully installed pip-23.1.2 wheel-0.40.0
.venv/bin/pip install -U -r requirements-dev.txt
Looking in indexes: https://pypi.devinfra.sentry.io/simple
Collecting black==22.10.0 (from -r requirements-dev.txt (line 6))
  Using cached https://pypi.devinfra.sentry.io/wheels/black-22.10.0-py3-none-any.whl (165 kB)
Collecting blinker==1.5 (from -r requirements-dev.txt (line 7))
  Using cached https://pypi.devinfra.sentry.io/wheels/blinker-1.5-py2.py3-none-any.whl (12 kB)
Collecting click==8.0.4 (from -r requirements-dev.txt (line 8))
  Using cached https://pypi.devinfra.sentry.io/wheels/click-8.0.4-py3-none-any.whl (97 kB)
Collecting flake8==6.0.0 (from -r requirements-dev.txt (line 9))
  Using cached https://pypi.devinfra.sentry.io/wheels/flake8-6.0.0-py2.py3-none-any.whl (57 kB)
ERROR: Could not find a version that satisfies the requirement confluent-kafka==1.9.2 (from versions: 2.1.1)
ERROR: No matching distribution found for confluent-kafka==1.9.2
```

Reply:
yep!  sentry uses `2.1.1` for instance (I think snuba team just standardized on this version)